plugins/registry.ts

```diff
@@ -187,20 +187,19 @@ export default class PluginRegistry {
     // Add a button to the channel intro message.
     // Accepts the following:
     // - icon - React element to use as the button's icon
-    // - action - a function called when the button is clicked, passed the channel and channel member as arguments
-    // - tooltipText - string or React element shown for tooltip appear on hover
+    // - action - a function called when the button is clicked
+    // - text - a localized string to use as the button's text
     registerChannelIntroButtonAction = reArg([
         'icon',
         'action',
-        'tooltipText',
     ], ({
         icon,
         action,
-        tooltipText,
+        text,
     }: {
         icon: ReactResolvable;
         action: PluginComponent['action'];
-        tooltipText: ReactResolvable;
+        text: string;
```

Comment:
I went with `text` being a plain string instead of a React component since that meant we don't need to pass a separate aria-label. On one hand, that means we have less props to pass around, but it does mean we lose some options around styling and formatting localized text